themes/default/content/blog/top-5-things-for-azure-devs-kubernetes-infrastructure/index.md

```diff
@@ -0,0 +1,566 @@
+- [Virtual networking](https://docs.microsoft.com/en-us/azure/aks/concepts-network) for pods in the cluster and other nodes using a peered virtual network. The network also provides ingress for applications with HTTP application routing.
+- Integration with development tools for Helm and a [Visual Studio Code extension for Kubernetes](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools).
+- Support for Docker images, including storage with [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/).
+
+## Create an AKS Cluster with the Azure Portal
+
+We'll use the Azure Portal to illustrate the steps to configure and deploy an AKS Cluster.
+
+**Step 1**: Create a Kubernetes Service.
+
+- Open the [Azure Portal](https:portal.azure.com) and select `Create Resource`.
+- Select `Containers` > `Kubernetes Service`
+- Select `Create`
+
+![Create Kubernetes service](./kubernetes_service.png)
+
+**Step 2**: Create an AKS cluster.
+
+For this example, we will configure several options in the `Basic` window but use defaults for most options.
+
+- In `Subscription`, Select an `Azure Subscription`
+- Create or select a [Resource Group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview)
+- In `Cluster details`, set the `Kubernetes cluster name`
+- Change the `Region` if needed
+
+Use the default values for `Primary node pool` and select `Next: node pools`.
+
+![Create AKS Cluster](./create_aks_cluster.png)
+
+**Step 3**: Configure node pools.
+
+A node pool is a logical grouping of nodes. Nodes in a pool can have different virtual machines, different Kubernetes versions, and other attributes. You can use different node pools for different purposes, such as grouping workflows, e.g., a node pool for production and one for dev or test.
+
+For this example, use the default values for node pools.
+
+![Node pools](./node_pools.png)
+
+**Step 4**: Configure authentication.
+
+You can authenticate, authorize, secure, and control access to Kubernetes clusters with:
+
+- [Kubernetes role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) (Kubernetes RBAC) to grant access to service accounts
+- [Azure Active Directory integration](https://docs.microsoft.com/en-us/azure/aks/managed-aad) that provides a multi-tenant, cloud-based directory and identity management service.
+
+To configure `Authentication` for this example, set the `Authentication method` to `System-assigned managed identity`. You can set `Role-based access control (RBAC)` to enabled, but it's unnecessary for the example. Use the default value for the `Node pool OS disk encryption` type.
+
+![Authentication](./authentication.png)
+
+**Step 5**: Configure networking
+
+AKS can use either `kubenet` or `Azure CNI` networking. Kubenet is the default configuration for AKS cluster creation.
```

Comment:
Note: I'm not changing these to bold because, while they are in the screenshot, they're actual products/things. I think it makes more sense to leave them as is since the text isn't explicitly referring to the UI.